docs/resources/Audit_Log.md

```diff
@@ -71,76 +71,80 @@ If no object is noted, there won't be a `changes` array in the entry, though oth
+| GUILD_SCHEDULED_EVENT_DELETE                | 102   | Event was cancelled                                       | [Guild Scheduled Event](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object)                                                      |
+| THREAD_CREATE                               | 110   | Thread was created in a channel                           | [Thread](#DOCS_RESOURCES_CHANNEL/thread-metadata-object)                                                                                         |
+| THREAD_UPDATE                               | 111   | Thread was updated                                        | [Thread](#DOCS_RESOURCES_CHANNEL/thread-metadata-object)                                                                                         |
+| THREAD_DELETE                               | 112   | Thread was deleted                                        | [Thread](#DOCS_RESOURCES_CHANNEL/thread-metadata-object)                                                                                         |
+| APPLICATION_COMMAND_PERMISSION_UPDATE       | 121   | Permissions were updated for a command                    | [Command Permission](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-permissions-object-application-command-permissions-structure)\* |
+| AUTO_MODERATION_RULE_CREATE                 | 140   | Auto Moderation rule was created                          | [Auto Moderation Rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object)                                                              |
+| AUTO_MODERATION_RULE_UPDATE                 | 141   | Auto Moderation rule was updated                          | [Auto Moderation Rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object)                                                              |
+| AUTO_MODERATION_RULE_DELETE                 | 142   | Auto Moderation rule was deleted                          | [Auto Moderation Rule](#DOCS_RESOURCES_AUTO_MODERATION/auto-moderation-rule-object)                                                              |
+| AUTO_MODERATION_BLOCK_MESSAGE               | 143   | Message was blocked by AutoMod (according to a rule)      |                                                                                                                                                  |
+| AUTO_MODERATION_FLAG_TO_CHANNEL             | 144   | Message was flagged by AutoMod (according to a rule)      |                                                                                                                                                  |
+| AUTO_MODERATION_USER_COMMUNICATION_DISABLED | 145   | Member was timed out by AutoMod (according to a rule)     |                                                                                                                                                  |
```

Comment:
```suggestion
| AUTO_MODERATION_BLOCK_MESSAGE               | 143   | Message was blocked by AutoMod                            |                                                                                                                                                  |
| AUTO_MODERATION_FLAG_TO_CHANNEL             | 144   | Message was flagged by AutoMod                            |                                                                                                                                                  |
| AUTO_MODERATION_USER_COMMUNICATION_DISABLED | 145   | Member was timed out by AutoMod                           |                                                                                                                                                  |
```

Reply:
i know you were staying consistent, but I think these can be taken out

Reply:
Yeah, kept them just for consistency, but they were somewhat redundant